contracts/Comptroller.sol

```diff
@@ -1348,6 +1315,18 @@ contract Comptroller is ComptrollerV6Storage, ComptrollerInterface, ComptrollerE
      * @return The amount of COMP which was NOT transferred to the user
      */
     function grantCompInternal(address user, uint amount) internal returns (uint) {
+        for (uint i = 0; i < allMarkets.length; ++i) {
+            address market = address(allMarkets[i]);
+
+            bool noOriginalSpeed = compBorrowSpeeds[market] == 0;
```

Comment:
This applies to both sides? We should be explicit about what's supposed to be happening here I think

Reply:
Yeah this applies to both sides because at the moment, compBorrowSpeeds[market] == compSupplySpeeds[market].

Reply:
GOLDSELLERUK/GDZIE GOLD